builtin/providers/nsone/resource_record.go

```diff
@@ -824,12 +339,20 @@ func RecordCreate(d *schema.ResourceData, meta interface{}) error {
 
 // RecordRead reads the DNS record from ns1
 func RecordRead(d *schema.ResourceData, meta interface{}) error {
-	client := meta.(*nsone.Client)
+	//client := meta.(*nsone.Client)
```

Comment:
@pashap: Whoops, looks like I left some debug code in here.

(Just in case you hadn't already noticed this yourself :-P)